tutorials/transfers/index.md

```diff
@@ -0,0 +1,271 @@
+
+    ![Accounts Tab](/images/tutorials/transfers/sending-funds/sending-funds-1.webp)
+
+2. Fill in the required fields:
+      1. **send from account** - select the account from which you want to send funds
+      2. **send to address** - enter the recipient's address
+      3. **amount** - enter the amount of funds you want to send
+      4. **transfer with account keep-alive checks** - enable this option if you want to keep the account alive
+      5. Click on the button **Make Transfer** to proceed:
+
+    ![Transfer Form](/images/tutorials/transfers/sending-funds/sending-funds-2.webp)
+
+    !!!warning
+        The transaction fees will be deducted from the leftover funds in your account. If you don't leave enough funds in your account for the fees, your transaction will fail due to "insufficient balance". Ensure to leave enough funds in your account to cover the transaction fees. If you want to send all the funds in your account, please refer to the [How to Send All of Your Funds Out of Your Account](https://support.polkadot.network/support/solutions/articles/65000170293){target=\_blank} guide.
+
+3. A pop-up window will appear to confirm the transfer. Click on the **Sign and Submit** button to authorize the transaction:
+
+    ![Transfer Confirmation](/images/tutorials/transfers/sending-funds/sending-funds-3.webp)
+
+4. To confirm that the transaction was successful, you can check the **Explorer** tab and search that the event **Transfer** has been emitted:
+
+    ![Explorer Tab](/images/tutorials/transfers/sending-funds/sending-funds-4.webp)
+
+### Extrinsics Verification
+
+The Polkadot Browser Extension is the entry point to the Web 3.0 Polkadot ecosystem, allowing you to connect with various compatible apps and websites. Therefore, verifying your extrinsic before signing them is crucial, especially when engaging with a new site or app.
+
+For further information on how to verify extrinsics, please refer to the [How Can I Verify What Extrinsic I'm Signing?](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-){target=\_blank} guide.
+
+### Keep Alive Checks
+
+In Polkadot, there are two main ways to transfer funds from one account to another:
+
+- Transfer `keep-alive` - (the default option) will not allow you to send an amount that would allow the sending account to be removed because it falls below the existential deposit.
+- Transfer `allow-death` - will allow you to send tokens regardless of the consequences. If the balance drops below the existential deposit, your account will be reaped. You may not want to keep the account alive, for example, if you are moving all of your funds to a different address. To turn off the keep-alive check, please visit the [Error: 'balances.KeepAlive'](https://support.polkadot.network/support/solutions/articles/65000169248){target=\_blank} guide.
```

Comment:
```suggestion
- Transfer `keep-alive` - (the default option) will not allow you to send an amount that would allow the sending account to be removed because it falls below the existential deposit
- Transfer `allow-death` - will allow you to send tokens regardless of the consequences. If the balance drops below the existential deposit, your account will be reaped. You may not want to keep the account alive, for example, if you are moving all of your funds to a different address. To turn off the keep-alive check, please visit the [Error: 'balances.KeepAlive'](https://support.polkadot.network/support/solutions/articles/65000169248){target=\_blank} guide
```